bin/commands/components/enable/index.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+#######################################################################
+# This program and the accompanying materials are made available
+# under the terms of the Eclipse Public License v2.0 which
+# accompanies this distribution, and is available at
+# https://www.eclipse.org/legal/epl-v20.html
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Copyright Contributors to the Zowe Project.
+#######################################################################
+
+require_zowe_yaml
+
+componentCfgPath=
+componentCfg=
+if [ -n "${ZWE_CLI_PARAMETER_HA_INSTANCE}" ]; then
+  componentCfgPath="haInstances.${ZWE_CLI_PARAMETER_HA_INSTANCE}.components.${ZWE_CLI_PARAMETER_COMPONENT_NAME}"
+else
+  componentCfgPath="components.${ZWE_CLI_PARAMETER_COMPONENT_NAME}"
+fi
+componentCfg=$(read_yaml "${ZWE_CLI_PARAMETER_CONFIG}" ".${componentCfgPath}")
```

Comment:
Instead of read component from zowe.yaml, think we should find whether the component is installed or not.

```
component_dir=$(find_component_directory "${ZWE_CLI_PARAMETER_COMPONENT_NAME}")
if [ -z "${component_dir}" ]; then
  # cannot find component
fi
```

Whether the component is defined in zowe.yaml doesn't matter here.